### DISCUSSION TITLE
Unable to integrate DAPR with open telemetry for AWS Xray exporter

### DISCUSSION THREAD

**Discussion by MGK91** [open-telemetry/opentelemetry-collector-contrib#14083] [Q&A]
Hi Team,

Greetings, can someone please help us with the configuration to export traces from DAPR to AWS Xray through Open telemetry exporter , DAPR and Open telemetry running on EKS

Reciever config is as follows 

      otlp:
        protocols:
          grpc:
            endpoint: 0.0.0.0:4317
          http:
            endpoint: 0.0.0.0:4318
      awsxray:
      awsxray/udp_endpoint:
          endpoint: "0.0.0.0:5678"
          transport: udp
      awsxray/proxy_server:
          proxy_server:
            endpoint: "0.0.0.0:1234"
            tls:
              insecure: true
            region: ap-south-1

Exporter config is as follows 

      awsxray:
        region: ap-south-1
        index_all_attributes: true
        resource_arn: "arn:aws:iam:xxxxxxxxxxx:role/xyzNodegroup" 
        role_arn: "arn:aws:iam::xxxxxxxxxxx:role/abc"

Service configuration as follows:

    service:
      extensions: [pprof, zpages, health_check]
      pipelines:
        traces:
          receivers: [otlp]
          # List your exporter here.
          exporters: [awsxray,logging]

DAPR Configuration as follows 
apiVersion: dapr.io/v1alpha1
kind: Configuration
metadata:
  name: appconfig
  namespace: dapr-test
spec:
  tracing:
    samplingRate: "1"
    zipkin:
      endpointAddress: "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans"

Tried configuring receiver as both otlp and awsxray and getting below mentioned error , Any help would be very much appreciated

Getting following error in DAPR 

2022/09/12 18:54:54 failed to send the request: Post "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans": dial tcp x.x.x.x:9411: connect: connection refused
2022/09/12 18:54:55 failed to send the request: Post "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans": dial tcp x.x.x.x:9411: connect: connection refused
2022/09/12 18:54:56 failed to send the request: Post "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans": dial tcp x.x.x.x:9411: connect: connection refused
2022/09/12 18:54:57 failed to send the request: Post "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans": dial tcp x.x.x.x:9411: connect: connection refused
2022/09/12 18:54:59 failed to send the request: Post "http://otel-collector.dapr-test.svc.cluster.local:9411/api/v2/spans": dial tcp x.x.x.x:9411: connect: connection refused

**Comment by dgoscn** [comment]
Hi @MGK91 sorry for the  dumb question. But, in your EKS which kind of SG are you added? The inbound rule is correctly applied for these ports? 

Which is generated in CloudTrail when the event that you make returns the connection refused?

Thanks
